assets/css/v2/pre_fare/subway_status.scss

```diff
@@ -0,0 +1,10 @@
+@import "../lcd_common_styles/subway_status.scss";
+
+.subway-status-footer {
+  background-color: #cccbc8 !important;
```

Comment:
I don't think `!important` is needed here because it will auto override whatever styles are in the import.

Reply:
oh gotcha, thanks!